src/Month.js

```diff
@@ -18,10 +18,10 @@ import DateContentRow from './DateContentRow'
 import Header from './Header'
 
 import { accessor, dateFormat } from './utils/propTypes'
-import { segStyle, inRange, sortEvents } from './utils/eventLevels'
+import { segStyle, within, sortEvents } from './utils/eventLevels'
 
 let eventsForWeek = (evts, start, end, props) =>
-  evts.filter(e => inRange(e, start, end, props))
+  evts.filter(e => within(e, start, end, props))
```

Comment:
the method `inRange()` operates only on the "day" part of the date, so when trying to figure out if an event is in the week range, we're not able to deduce if the event end date is great than the week start date. 

this new method checks for the whole date if it's within the given range.